script.js

```diff
@@ -2,57 +2,49 @@ let startBtn = document.querySelector("#startButton");
 startBtn.addEventListener("click", playGame);
 //ash says we can use this as a global object
 let gameState = ["", "", "", "", "", "", "", "", ""];
-function incrementCurrentPlayer(currentPlayer) {
-    currentPlayer++;
-    return currentPlayer;
-}
-function decideTurn(currentPlayer, playerMarker) {
-    if (currentPlayer % 2 == 0) {
-        playerMarker = 'o'
-    } else {
-        playerMarker = 'x'
+
+function decideTurn(currentPlayer) {
+    if (currentPlayer === 'o') {
+        return 'x';
+    }   else  {
+        return  'o';
     }
-    return playerMarker;
 }
-function addToScreen(tile, playerMarker) {
-    tile.innerHTML = playerMarker;
+function addToScreen(tile, currentPlayer) {
+    tile.innerHTML = currentPlayer;
 }
-function alterGameState(gameState, clicked, playerMarker) {
+function alterGameState(gameState, clicked, currentPlayer) {
     if(gameState[clicked] == "") {
-        gameState[clicked] = playerMarker;
+        gameState[clicked] = currentPlayer;
         return true;
     } else {
         return false;
     }
```

Comment:
No need for the else condition when returning in the if.

Take a look at this example - it achieves the exact same thing but is a bit simpler
```js
function alterGameState(gameState, clicked, currentPlayer) {
    if(gameState[clicked] == "") {
        gameState[clicked] = currentPlayer;
        return true;
    } 
    return false;
}
```